drivers/sensor/lps22hb/Kconfig

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2017 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig LPS22HB
+	bool
+	prompt "LPS22HB pressure and temperature"
+	depends on SENSOR && I2C
+	default n
+	help
+	  Enable driver for LPS22HB I2C-based pressure and temperature
+	  sensor.
+
+config LPS22HB_DEV_NAME
+	string
+	prompt "Device name"
+	default "LPS22HB"
+	depends on LPS22HB
+	help
+	 Device name used for LPS22HB sensor identification.
+
+config LPS22HB_I2C_ADDR
+	hex
+	prompt "I2C address"
+	depends on LPS22HB
+	default "0x5D"
+	help
+	  I2C address of the LPS22HB sensor.
+	  Use 0x5C if the SA0 pin is pulled to GND or 0x5D if the SA0
+	  pin is pulled to VDD.
+
+config LPS22HB_I2C_MASTER_DEV_NAME
+	string
+	prompt "I2C master where LPS22HB is connected"
+	depends on LPS22HB
+	default "I2C_2"
```

Comment:
Ok. I set this since built in sensor is using this configuration in Disco board I just pushed.
But I can change it and configure address in the board config files